index.js

```diff
@@ -7,8 +7,10 @@ const D2LHypermediaRels = {
 	color: 'https://api.brightspace.com/rels/color',
 	content: 'https://api.brightspace.com/rels/content',
 	courseOfferingInfoPage: 'https://api.brightspace.com/rels/course-offering-info-page',
+	defaultSearch: 'https://api.brightspace.com/rels/default-search',
 	departments: 'https://api.brightspace.com/rels/departments',
 	displayName: 'https://api.brightspace.com/rels/display-name',
+	enrollments: 'https://api.brightspace.com/rels/enrollments',
```

Comment:
Huh, crazy that this wasn't in here before! I guess we mostly just used the the action by name instead.